lib/prx_auth/rails/ext/controller.rb

```diff
@@ -52,7 +52,10 @@ def current_user
       end
 
       def current_user_info
-        session[PRX_USER_INFO_SESSION_KEY] ||= fetch_userinfo
+        session[PRX_USER_INFO_SESSION_KEY] ||= begin
+          info = fetch_userinfo
+          info.slice('name', 'preferred_username', 'email', 'image_href', 'apps')
```

Comment:
Ever since ID was converted to `hal_builder` (and more verbose HAL output) ... we've been storing some rather large hashes in the session!

I've minimized them here, but even so, we should probably look into a better strategy for caching these.  (Redis?  Localstorage?  Anything else?).  Someday.